python/custom/templates/PerLineMcdc/main.html.template

```diff
@@ -0,0 +1,113 @@
+{#- 6 spaces (3 levels) indent to match #}
```

Comment:
Remove this for now; we'll add it it back in a later PR

Reply:
Removed in b47303ebd7cd9ac952b379fb59fcb96cd0b4c57d